### PR TITLE
翻译完成 community/roadmap.md, community/style_guide.md, deploy/s3.md, install/_index.yaml, install/pip.html and install/source_windows.md

### LINKED ISSUE #122: community/roadmap.md, community/style_guide.md, deploy/s3.md, install/_index.yaml, install/pip.html and install/source_windows.md
* Markdown 文件：
    * [x] [community/roadmap.md](https://github.com/xitu/tensorflow-docs/blob/zh-hans/community/roadmap.md)
    * [x] [community/style_guide.md](https://github.com/xitu/tensorflow-docs/blob/zh-hans/community/style_guide.md)
    * [x] [deploy/s3.md](https://github.com/xitu/tensorflow-docs/blob/zh-hans/deploy/s3.md)
    * [x] [install/_index.yaml](https://github.com/xitu/tensorflow-docs/blob/zh-hans/install/_index.yaml)
    * [x] [install/pip.html](https://github.com/xitu/tensorflow-docs/blob/zh-hans/install/pip.html)
    * [x] [install/source_windows.md](https://github.com/xitu/tensorflow-docs/blob/zh-hans/install/source_windows.md)
  * PR 地址：https://github.com/xitu/tensorflow-docs/pull/134
  **注意**：这些内容以几个 PR 提交均可，看你的安排。

  ----

  * 翻译时间：10 天
  * 校对时间：3 天
  * 翻译积分：10 分
  * 校对积分：2-5 分
  * [积分有什么用](https://github.com/xitu/gold-miner/wiki)

### PR DESCRIPTION
**Pull request by sisibeloved**
resolve #122

### CODE REVIEW

**Comment by haiyang-tju** [comment]
校对认领 @sisibeloved @leviding @fanyijihua

File: community/roadmap.md [hunk 1]
```diff
@@ -2,141 +2,140 @@
...
-#### Reference models
+#### 参考模型
 
-* Building out a set of [models](https://github.com/tensorflow/models/tree/master/official) across image recognition, object detection, speech, translation, recommendation, and reinforcement learning that demonstrate best practices and serve as a starting point for high-performance model development.
-* A growing set of high-performance [Cloud TPU reference models](https://github.com/tensorflow/tpu).
+* 内建了一组跨图像识别、对象检测、语音、翻译、推荐和增强学习的[模型](https://github.com/tensorflow/models/tree/master/official)，这些模型能够展示最佳效果，并作为高性能模型开发的起点。
+* 扩大了高性能[云 TPU 参考模型](https://github.com/tensorflow/tpu)的规模。
 
-#### Contrib
+#### 软件库
```
Comment (haiyang-tju) [inline community/roadmap.md]:
```suggestion
#### 扩展包
```

File: community/roadmap.md [hunk 2]
```diff
@@ -2,141 +2,140 @@
...
 
-* Deprecate parts of `tf.contrib` where preferred implementations exist outside of `tf.contrib`.
-* As much as possible, move large projects inside `tf.contrib` to separate repositories.
-* The `tf.contrib` module will be discontinued in its current form with TensorFlow 2.0. Experimental development will happen in other repositories in the future.
+* 废弃 `tf.contrib` 中有更好的外部实现的部分。
+* 尽可能地讲 `tf.contrib` 中的大型项目移到单独的仓库中。
```
Comment (haiyang-tju) [inline community/roadmap.md]:
```suggestion
* 尽可能地将 `tf.contrib` 中的大型项目移到单独的仓库中。
```

File: community/roadmap.md [hunk 3]
```diff
@@ -2,141 +2,140 @@
...
+* 添加对 Edge TPU 和 TPU AIY 主板的支持。
+* 更加易用的文档和教程。
 
 #### TensorFlow.js
 
-* Improve performance of TensorFlow.js in the browser; Implement prototype using compute shaders or WebGPU, when available; Improve CPU performance, implement SIMD+ Web Assembly support when available.
-* Expand support for importing TensorFlow SavedModels and Keras models, with focus on audio and text-based models.
-* Release a new tfjs-data API for efficient data input pipelines, and a new tfjs-vis library for interactive model visualizations during browser training.
-* For server-side TensorFlow.js using Node - improve parity with native TensorFlow ops and model formats by exposing all TensorFlow ops; Add async mode support using libuv.
+* 提高 TensorFlow.js 在浏览器中的性能；使用计算着色器或 WebGPU 实现原型；提高 CPU 性能，提供 SIMD+ Web 程序集支持。
```
Comment (haiyang-tju) [inline community/roadmap.md:58]:
```suggestion
* 提高 TensorFlow.js 在浏览器中的性能；使用计算着色器或 WebGPU 实现原型；提高 CPU 性能，提供 SIMD + Web 程序集支持。
```

File: community/roadmap.md [hunk 4]
```diff
@@ -2,141 +2,140 @@
...
-* CUDA 10 integration (plan to skip CUDA 9.2 as it has minimal advantages to CUDA 9.0 when using the same version of cuDNN).
-* Optimizations for DGX-2.
+* 修改通用设计，简化固定精度 API。
+* 完成 TensorRT API 并移入核心模块。
+* TensorRT 现在支持 SavedModel 和 TF Serving 了。
+* 集成 CUDA 10（计划跳过 CUDA 9.2 版本因为跟 CUDA 9.0 相比它在相同版本的 cuDNN 下的改进太小了）。
+* 优化对 DGX-2 的支持。
 
-#### Cloud TPUs and Cloud TPU Pods
+#### 云 TPU 和云 TPU Pod
```
Comment (haiyang-tju) [inline community/roadmap.md]:
```suggestion
#### 云 TPU 和云 TPU Pods
```

File: community/roadmap.md [hunk 5]
```diff
@@ -2,141 +2,140 @@
...
+* 通过 MKL 为 SkyLake 系列处理器提供 Int8 支持。
+* 通过 MKL 提供更快的 3D 操作。
+* 动态加载专为 SIMD 优化的内核。
+* MKL 的 Linux 和 Windows 支持。
 
-### Other packages
+### 其它模块
 
-#### TensorFlow Probability
+#### TensorFlow 概率
```
Comment (haiyang-tju) [inline community/roadmap.md]:
```suggestion
#### TensorFlow 概率编程工具箱
```

File: community/style_guide.md [hunk 6]
```diff
@@ -50,13 +50,13 @@ srcs_version = "PY2AND3",
 
 ## Tensor
 
-* 在假设 Tensor 的第一维度是 batche 维度的情况下对 batches 操作的处理函数。
+* 批处理操作通常假设 Tensor 的第一个维度是批处理维度。
 
-* In most models the *last dimension* is the number of channels.
+* 在大多数模型中**最后一个维度**是信道的数量。
```
Comment (haiyang-tju) [inline community/style_guide.md]:
```suggestion
* 在大多数模型中**最后一个维度**是通道的数量。
```

**Review by haiyang-tju — COMMENTED** [review]
校对完成，这个翻译的非常好~~

@sisibeloved @leviding

**Comment by leviding** [comment]
@haiyang-tju 赞 @sisibeloved 可以修改啦

**Review by leviding — APPROVED** [review]

**Comment by leviding** [comment]
辛苦啦各位！

**Pull request merged**